addons/field_rations/functions/fnc_update.sqf

```diff
@@ -66,7 +67,7 @@ _player setVariable [QGVAR(hunger), _hunger, _doSync];
 [_player, _thirst, _hunger] call FUNC(handleEffects);
 
 // Handle showing/updating or hiding of HUD
-if (_thirst > GVAR(hudShowLevel) || {_hunger > GVAR(hudShowLevel)} || {GVAR(hudInteractionHover)}) then {
+if (!ACEGVAR(common,OldIsCamera) && {_thirst > GVAR(hudShowLevel) || {_hunger > GVAR(hudShowLevel)} || {GVAR(hudInteractionHover)}}) then {
```

Comment:
Shouldn't we just use feature camera from CBA? I forget how this variable is linked to it.

Reply:
This is variable is true if a CBA registered feature camera is active, false otherwise. [reference](https://github.com/acemod/ACE3/blob/master/addons/common/XEH_postInit.sqf#L298)

Reply:
That's just for backwards compatibility though, should we really use that variable here?

Reply:
Could change this to either:

- `call CBA_fnc_getActiveFeatureCamera == ""`
- Add a "featureCamera" event that similarly keeps a variable updated

I think the second might be a bit better since this runs every second.

Reply:
I think this is fine, checking a gvar is faster than doing the _fnc call

Reply:
For that, it's fine to use the ACE3 one. But that could and probably should be part of CBA really.

Reply:
cba also has var `cba_events_oldFeatureCamera`
but that's a string instead of a bool

Reply:
cba also has `cba_events_oldFeatureCamera`
but that's a string instead of a bool